### PR TITLE
New Instrument Cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
cal source files are noted in each cal csv

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/FLORTD/CGINS-FLORTD-01102__20201110.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+1102,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1102_Calibration_Files_2020-11-10.zip > BBFL2W-1102 Charsheets.pdf
+1102,CC_dark_counts_cdom,42,
+1102,CC_dark_counts_chlorophyll_a,48,
+1102,CC_dark_counts_volume_scatter,40,
+1102,CC_depolarization_ratio,0.039,Constant
+1102,CC_measurement_wavelength,700,Constant
+1102,CC_scale_factor_cdom,0.0657,
+1102,CC_scale_factor_chlorophyll_a,0.0106,
+1102,CC_scale_factor_volume_scatter,2.016E-06,
+1102,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01102__20201110.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01105__20201105.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1105,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1105_Calibration_Files_2020-11-05.zip > BBFL2W-1105 Charsheets.pdf
+1105,CC_dark_counts_cdom,42,
+1105,CC_dark_counts_chlorophyll_a,52,
+1105,CC_dark_counts_volume_scatter,47,
+1105,CC_depolarization_ratio,0.039,Constant
+1105,CC_measurement_wavelength,700,Constant
+1105,CC_scale_factor_cdom,0.0764,
+1105,CC_scale_factor_chlorophyll_a,0.0114,
+1105,CC_scale_factor_volume_scatter,2.377E-06,
+1105,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01105__20201105.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01107__20201113.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+1107,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1107_Calibration_Files_2020-11-13.zip > BBFL2W-1107 Charsheets.pdf
+1107,CC_dark_counts_cdom,42,
+1107,CC_dark_counts_chlorophyll_a,52,
+1107,CC_dark_counts_volume_scatter,47,
+1107,CC_depolarization_ratio,0.039,Constant
+1107,CC_measurement_wavelength,700,Constant
+1107,CC_scale_factor_cdom,0.0779,
+1107,CC_scale_factor_chlorophyll_a,0.0121,
+1107,CC_scale_factor_volume_scatter,2.377E-06,
+1107,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01107__20201113.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01116__20201110.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1116,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1116_Calibration_Files_2020-11-10.zip > BBFL2W-1116 Charsheets.pdf
+1116,CC_dark_counts_cdom,44,
+1116,CC_dark_counts_chlorophyll_a,47,
+1116,CC_dark_counts_volume_scatter,46,
+1116,CC_depolarization_ratio,0.039,Constant
+1116,CC_measurement_wavelength,700,Constant
+1116,CC_scale_factor_cdom,0.0642,
+1116,CC_scale_factor_chlorophyll_a,0.0112,
+1116,CC_scale_factor_volume_scatter,2.190E-06,
+1116,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01116__20201110.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01213__20201112.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+1213,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1213_Calibration_Files_2020-11-12.zip > BBFL2W-1213 Charsheets.pdf
+1213,CC_dark_counts_cdom,46,
+1213,CC_dark_counts_chlorophyll_a,46,
+1213,CC_dark_counts_volume_scatter,47,
+1213,CC_depolarization_ratio,0.039,Constant
+1213,CC_measurement_wavelength,700,Constant
+1213,CC_scale_factor_cdom,0.0909,
+1213,CC_scale_factor_chlorophyll_a,0.0118,
+1213,CC_scale_factor_volume_scatter,2.013E-06,
+1213,CC_scattering_angle,124,Constant
```
Comment (cdobs) [inline calibration/FLORTD/CGINS-FLORTD-01213__20201112.csv:11]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18353__20201106.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18353,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_18353_Configuration_Summary_2020-11-06.pdf
+18353,CC_scale_factor2,0.45,constant
+18353,CC_scale_factor3,0.45,constant
+18353,CC_scale_factor4,0.45,constant
+18353,CC_bin_size,0,in cm; return from vendor - not allocated yet
+18353,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+18353,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-18353__20201106.csv:8]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18446__20201103.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18446,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_18446_Configuration_Summary_2020-11-03.pdf
+18446,CC_scale_factor2,0.45,constant
+18446,CC_scale_factor3,0.45,constant
+18446,CC_scale_factor4,0.45,constant
+18446,CC_bin_size,0,in cm; return from vendor - not allocated yet
+18446,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+18446,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-18446__20201103.csv:8]: Looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21654__20200916.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21654,CC_scale_factor1,0.45,constant; source_file ADCPS-N_WHLS75-1500-I_SN_21654_Configuration_Summary_2020-09-16.pdf
+21654,CC_scale_factor2,0.45,constant
+21654,CC_scale_factor3,0.45,constant
+21654,CC_scale_factor4,0.45,constant
+21654,CC_bin_size,0,in cm; return from vendor - not allocated yet
+21654,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+21654,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSN/CGINS-ADCPSN-21654__20200916.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20201026.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
...273.48, 274.28, 275.08, 275.88, 276.68, 277.48, 278.28, 279.08, 279.88, 280.69, 281.49, 282.29, 283.09, 283.89, 284.69, 285.49, 286.29, 287.09, 287.89, 288.69, 289.49, 290.29, 291.09, 291.89, 292.69, 293.49, 294.29, 295.09, 295.89, 296.69, 297.49, 298.29, 299.09, 299.89, 300.69, 301.49, 302.29, 303.09, 303.89, 304.68, 305.48, 306.28, 307.08, 307.88, 308.68, 309.48, 310.28, 311.08, 311.88, 312.68, 313.47, 314.27, 315.07, 315.87, 316.67, 317.47, 318.27, 319.06, 319.86, 320.66, 321.46, 322.26, 323.05, 323.85, 324.65, 325.45, 326.24, 327.04, 327.84, 328.63, 329.43, 330.23, 331.03, 331.82, 332.62, 333.41, 334.21, 335.01, 335.8, 336.6, 337.39, 338.19, 338.99, 339.78, 340.58, 341.37, 342.17, 342.96, 343.76, 344.55, 345.34, 346.14, 346.93, 347.73, 348.52, 349.31, 350.11, 350.9, 351.69, 352.49, 353.28, 354.07, 354.87, 355.66, 356.45, 357.24, 358.03, 358.83, 359.62, 360.41, 361.2, 361.99, 362.78, 363.57, 364.36, 365.15, 365.94, 366.73, 367.52, 368.31, 369.1, 369.89, 370.68, 371.47, 372.26, 373.04, 373.83, 374.62, 375.41, 376.2, 376.98, 377.77, 378.56, 379.34, 380.13, 380.91, 381.7, 382.49, 383.27, 384.06, 384.84, 385.63, 386.41, 387.2, 387.98, 388.76, 389.55, 390.33, 391.11, 391.9, 392.68]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01089__20201026.csv:8]: Spot check - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01090__20200904.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
...272.23, 273.03, 273.84, 274.64, 275.44, 276.25, 277.05, 277.85, 278.66, 279.46, 280.26, 281.07, 281.87, 282.68, 283.48, 284.28, 285.09, 285.89, 286.7, 287.5, 288.31, 289.11, 289.92, 290.72, 291.52, 292.33, 293.13, 293.94, 294.74, 295.55, 296.35, 297.16, 297.97, 298.77, 299.58, 300.38, 301.19, 301.99, 302.8, 303.6, 304.41, 305.21, 306.02, 306.83, 307.63, 308.44, 309.24, 310.05, 310.85, 311.66, 312.47, 313.27, 314.08, 314.88, 315.69, 316.49, 317.3, 318.11, 318.91, 319.72, 320.52, 321.33, 322.14, 322.94, 323.75, 324.55, 325.36, 326.16, 326.97, 327.78, 328.58, 329.39, 330.19, 331, 331.8, 332.61, 333.42, 334.22, 335.03, 335.83, 336.64, 337.44, 338.25, 339.05, 339.86, 340.66, 341.47, 342.27, 343.08, 343.88, 344.69, 345.49, 346.3, 347.1, 347.91, 348.71, 349.52, 350.32, 351.13, 351.93, 352.74, 353.54, 354.34, 355.15, 355.95, 356.76, 357.56, 358.36, 359.17, 359.97, 360.77, 361.58, 362.38, 363.18, 363.99, 364.79, 365.59, 366.4, 367.2, 368, 368.8, 369.61, 370.41, 371.21, 372.01, 372.81, 373.61, 374.42, 375.22, 376.02, 376.82, 377.62, 378.42, 379.22, 380.02, 380.82, 381.62, 382.43, 383.23, 384.03, 384.83, 385.62, 386.42, 387.22, 388.02, 388.82, 389.62, 390.42, 391.22, 392.02, 392.81, 393.61]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01090__20200904.csv:8]: Spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20201116.csv [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+ACS-254,CC_tbins,"[0.970387, 1.193668, 2.465714, 3.456000, 4.476875, 5.525385, 6.458261, 7.512308, 8.438400, 9.508182, 10.441000, 11.536087, 12.462083, 13.511250, 14.473750, 15.515185, 16.476522, 17.482000, 18.458462, 19.528333, 20.481154, 21.498400, 22.477083, 23.501923, 24.528261, 25.476538, 26.493600, 27.492308, 28.518846, 29.472500, 30.505417, 31.464839, 32.496500, 33.533333, 34.474000]",
+ACS-254,CC_taarray,SheetRef:CC_taarray,
+ACS-254,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20201116.csv:9]: Spot check, looks good

**Review by cdobs — COMMENTED** [review]
Reviewed everything except the optaa tcarray.ext files. I don't remember what is required to review those. 

Also have a question about the formatting of the DOSTAD cal csv. Left a comment on those.

File: calibration/DOSTAD/CGINS-DOSTAD-00502__20201004.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+502,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_502_Multipoint_Calibration_2020-10-04.pdf
+502,CC_csv,"[2.65279E-03, 1.13001E-04, 2.24904E-06, 9.12452E+01, -1.45210E-01, -2.24209E+01, 1.81645E+00]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/DOSTAD/CGINS-DOSTAD-00502__20201004.csv:3]: Format looks good after notes column changes

File: calibration/DOSTAD/CGINS-DOSTAD-00422__20201004.csv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+422,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_422_Multipoint_Calibration_2020-10-04.pdf
+422,CC_csv,"[2.57880E-03, 1.08179E-04, 2.29233E-06, 1.81996E+02, -2.80249E-01, -4.74749E+01, 3.63161E+00]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/DOSTAD/CGINS-DOSTAD-00422__20201004.csv:3]: Format looks good after notes column changes

File: calibration/DOSTAD/CGINS-DOSTAD-00382__20201004.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+382,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_382_Multipoint_Calibration_2020-10-04.pdf
+382,CC_csv,"[2.55735E-03, 1.07584E-04, 2.26331E-06, 1.47379E+02, -2.25835E-01, -3.72776E+01, 2.95136E+00]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/DOSTAD/CGINS-DOSTAD-00382__20201004.csv:3]: Format looks good after notes column changes

File: calibration/DOSTAD/CGINS-DOSTAD-00131__20201004.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+131,CC_conc_coef,"[0.00000E+00, 1.00000E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_131_Multipoint_Calibration_2020-10-04.pdf
+131,CC_csv,"[2.64282E-03, 1.10041E-04, 2.28331E-06, 1.77157E+02, -2.43014E-01, -4.15779E+01, 3.51030E+00]",
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/DOSTAD/CGINS-DOSTAD-00131__20201004.csv:3]: Format looks good after notes column changes

**Review by cdobs — APPROVED** [review]
Looks good to me!

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20201026.csv (hunk 9 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01089__20201026.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01090__20200904.csv (hunk 10 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01090__20200904.csv:8]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20201116.csv (hunk 11 above)
Comment (snwhite) [reply calibration/OPTAAD/CGINS-OPTAAD-00254__20201116.csv:9]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20201116__CC_tcarray.ext [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+-0.011799,-0.009474,-0.006378,-0.005959,-0.005652,-0.005173,-0.004464,-0.004273,-0.003681,-0.003417,-0.002615,-0.002453,-0.001915,-0.001597,-0.001177,-0.000869,-0.000688,-0.000562,-0.000335,-0.000575,-0.000303,-0.000215,-0.000175,-0.000159,-0.000275,0,-0.000151,-0.000156,-0.000123,-0.000185,-0.000161,-0.000194,-0.000611,-0.000781,-0.001141
+-0.011245,-0.009033,-0.005804,-0.005332,-0.005065,-0.004665,-0.004227,-0.003823,-0.003292,-0.002988,-0.002377,-0.002158,-0.00178,-0.001626,-0.001192,-0.000808,-0.000676,-0.000437,-0.000243,-0.00041,-0.000108,-0.000134,-4.8e-05,-7.7e-05,-0.000166,0,4.9e-05,0.000134,-0.000114,-9e-05,-0.000217,-0.000179,-0.000597,-0.000878,-0.001132
+-0.013008,-0.010682,-0.007078,-0.006515,-0.006118,-0.005785,-0.005018,-0.004724,-0.004092,-0.003939,-0.003151,-0.002926,-0.002514,-0.002166,-0.001723,-0.001317,-0.001057,-0.000968,-0.000652,-0.000877,-0.000349,-0.000378,-0.000192,-0.000257,-0.000249,0,-5.4e-05,-5.4e-05,-0.000173,-0.000191,-0.000184,-0.000179,-0.000677,-0.001031,-0.00148
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20201116__CC_tcarray.ext:85]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20201116__CC_taarray.ext [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+-0.008101,-0.00642,-0.004286,-0.004064,-0.003797,-0.003674,-0.003385,-0.002991,-0.002943,-0.00262,-0.002467,-0.002179,-0.001921,-0.001759,-0.001583,-0.001294,-0.001253,-0.000962,-0.000744,-0.000684,-0.000402,-0.000291,-0.000167,-2e-06,3.1e-05,0,-1.6e-05,-8.5e-05,-6.2e-05,-0.000109,-0.000213,-0.000432,-0.000505,-0.000869,-0.001032
+-0.008296,-0.006621,-0.004466,-0.00426,-0.003874,-0.00385,-0.003508,-0.003134,-0.003084,-0.002716,-0.002588,-0.002309,-0.001998,-0.001852,-0.001691,-0.001384,-0.00139,-0.00097,-0.000892,-0.000775,-0.000483,-0.000365,-0.000236,-1.8e-05,-5.3e-05,0,-1.3e-05,-7.8e-05,-9e-06,-0.000118,-0.000217,-0.000452,-0.000489,-0.000897,-0.001058
+-0.008503,-0.006766,-0.00453,-0.004333,-0.004029,-0.003893,-0.003558,-0.003221,-0.00315,-0.002787,-0.002609,-0.002352,-0.002079,-0.001896,-0.001676,-0.001358,-0.001386,-0.001053,-0.000852,-0.000737,-0.000509,-0.000311,-0.000171,-3.9e-05,5.1e-05,0,1.6e-05,-6.1e-05,5.1e-05,-9.6e-05,-0.000167,-0.000392,-0.000392,-0.000751,-0.000962
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20201116__CC_taarray.ext:85]: Looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/ADCPSL/CGINS-ADCPSL-20503__20201117.csv [hunk 18]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20503,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_20503_Configuration_Summary_2020-11-17.pdf
+20503,CC_scale_factor2,0.45,constant
+20503,CC_scale_factor3,0.45,constant
+20503,CC_scale_factor4,0.45,constant
+20503,CC_bin_size,0,in cm; return from vendor - not allocated yet
+20503,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+20503,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/ADCPSL/CGINS-ADCPSL-20503__20201117.csv:8]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good!

**Pull request merged**